src/routes/+layout.svelte

```diff
@@ -90,14 +90,73 @@
 <style>
 	/* === Variables === */
 	:root {
+		--orange-color: #fa796e;
+
+		--border-size: 1px;
+
+		--main-font: 'Akrobat-black';
+		--secondary-font: Arial;
+
+		--padding: 8px;
+		--padding-0_25x: calc(var(--padding) / 4);
+		--padding-0_5x: calc(var(--padding) / 2);
+		--padding-1_5x: calc(1.5 * var(--padding));
+		--padding-2x: calc(2 * var(--padding));
+		--padding-3x: calc(3 * var(--padding));
+		--padding-4x: calc(4 * var(--padding));
+		--padding-6x: calc(6 * var(--padding));
+		--padding-8x: calc(8 * var(--padding));
+		--card-background-contrast: var(--main-color);
+		--card-background: white;
+	}
+
+	:root[theme='light'] {
+		--main-color: #283e95;
+		--stake-text-color: black;
+
+		--title-color: black;
+		--border-color: #454545;
+		--page-background: #fcfffd;
+		--background-color: #fcfffd;
+		--qr-code-background: #fcfffd;
+
+		--input-color: #fcfffd;
+
+		--main-button-text-color: white;
+
+		--input-border: 2px solid var(--border-color);
+
+		--text-color: rgb(127 127 127);
+		--faq-color: black;
+
+		--svg-fill-color: #000000;
+		--svg-stroke-color: #000000;
+
+		--sns-selected-button-color: rgb(107 180 249 / 50%);
+	}
+
+	:root[theme='dark'] {
+		--svg-fill-color: #ffffff; /* White for dark theme */
+		--svg-stroke-color: #ffffff; /* Optional: Stroke color */
```

Comment:
unnecessary variable 
However we could with --svg-opposite-color to display a nice gradient on the open chat logo